components/post_view/post_reaction/post_reaction.tsx

```diff
@@ -42,6 +42,11 @@ type State = {
     showEmojiPicker: boolean;
 }
 
+// An emoji picker in the RHS isn't constrained by the RHS, so it just needs space to fit
+// the emoji picker itself
+const RHS_SPACE_REQUIRED_ABOVE = 420;
+const RHS_SPACE_REQUIRED_BELOW = 420;
+
```

Comment:
this should still live in the actual component that uses it. I agree that exporting it as a class member is not ideal, but you could define a constant and export that instead, since those values are static that should not be a problem.

```suggestion
```